website/app/components/dynamic-template.js

```diff
@@ -1,61 +1,68 @@
-/* eslint-disable ember/no-classic-components, ember/no-classic-classes, prettier/prettier, no-console */
-
-import Component from '@ember/component';
-import GlimmerComponent from '@glimmer/component';
-import layout from './dynamic-template';
-
-import { computed } from '@ember/object';
+import { setComponentTemplate } from '@ember/component';
 import { getOwner } from '@ember/application';
 import { compileTemplate } from '@ember/template-compilation';
+import { importSync } from '@embroider/macros';
+import Component from '@glimmer/component';
 
 let templateOwnerMap = new Map();
-let templateId = 0;
 
-export default Component.extend({
-  tagName: '',
-  layout,
-  init() {
-    this._super(...arguments);
+export default class DynamicTemplate extends Component {
```

Comment:
[nit] to keep the implementation in sync ;)

```suggestion
export default class DynamicTemplateComponent extends Component {
```

Reply:
Actually this is now antipattern, because in the future you would be importing things like you name them eg.g. `<HdsAlert>` backing class has to be named `HdsAlert` not `HdsAlertComponent` otherwise it wold look strange